### DISCUSSION TITLE
My Question

### DISCUSSION THREAD

**Discussion by Pulkit068** [community/community#37513] [Other Feature Feedback, Questions, & Ideas]
How to make write code in github on android

**Comment by GiMMC** [comment]
Try to code with visual studio code at commit to github/ create new file and code inside it

**Reply by Pulkit068** [reply]
On android

**Comment by ossanna16** [comment]
Hi there @Pulkit068! Can you please elaborate further on your question so we can help?

**Reply by Pulkit068** [reply]
On android

**Comment by jfullstackdev** [comment]
not very sure, maybe like this 

https://www.youtube.com/watch?v=PoQWJNp-5s4

if it can push to GitHub, like the regular VS Code, then you're good to go
